client/index.js

```diff
@@ -1,10 +1,13 @@
 import React from 'react'
 import { render } from 'react-dom'
+import { Router, Route, Link, hashHistory, IndexRoute } from 'react-router'
 import { createStore, applyMiddleware, compose } from 'redux'
 import { Provider } from 'react-redux'
 
-import reducer from './reducer'
+import reducer from './reducers/landingReducer'
```

Comment:
this doesn't look right ... we want to use `reducer.js` and either put everyones reducers into that file, or have several files and use combine reducers.

I recommend we just use one reducer file given where we're at at the moment